config/crd/bases/oadp.openshift.io_nonadminrestores.yaml

```diff
@@ -0,0 +1,664 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: nonadminrestores.oadp.openshift.io
+spec:
+  group: oadp.openshift.io
+  names:
+    kind: NonAdminRestore
+    listKind: NonAdminRestoreList
+    plural: nonadminrestores
+    shortNames:
+    - nar
+    singular: nonadminrestore
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: NonAdminRestore is the Schema for the nonadminrestores API
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: NonAdminRestoreSpec defines the desired state of NonAdminRestore
+            properties:
+              restoreSpec:
+                description: restoreSpec defines the specification for a Velero restore.
+                properties:
+                  backupName:
+                    description: |-
+                      BackupName is the unique name of the Velero backup to restore
```

Comment:
could change here to NonAdminBackup to not confuse user

Reply:
Should during **ValidateRestoreSpec** step there be a check if actually NonAdminBackup from which restore will happen exists in the users NS?